frontend/src/layout/onboarding/onboarding.js

```diff
@@ -0,0 +1,162 @@
+import React, { useState, useEffect, useRef } from 'react'
+import { Popover, Button, Checkbox, Badge, Modal } from 'antd'
+import { useValues, useActions } from 'kea'
+import { actionsModel } from '~/models/actionsModel'
+import { Loading } from 'lib/utils'
+import { StarOutlined, StarFilled } from '@ant-design/icons'
+import { Link } from 'lib/components/Link'
+import FunnelImage from '../_assets/funnel_with_text.png'
+import ActionImage from '../_assets/actions.png'
+import TrendImage from '../_assets/trend.png'
+import { onboardingLogic, TourType } from './onboardingLogic'
+import { userLogic } from 'scenes/userLogic'
+import _ from 'lodash'
+import api from 'lib/api'
+
+export function OnboardingWidget() {
+    const contentRef = useRef()
+    const { actions, actionsLoading } = useValues(actionsModel)
+    const [instructionalModal, setInstructionalModal] = useState(false)
+    const { user } = useValues(userLogic)
+    const { loadUser } = useActions(userLogic)
+    const { tourType, checked } = useValues(onboardingLogic({ user }))
+    const { setTourActive, setTourType, updateOnboardingInitial } = useActions(onboardingLogic)
+    const [visible, setVisible] = useState(user.onboarding.initial ? true : false)
+
+    const unfinishedCount = _.filter(checked, (isChecked) => !isChecked).length
+
+    function closePopup() {
+        if (user.onboarding.initial) updateOnboardingInitial(false)
```

Comment:
`updateOnboardingInitial` looks like it would benefit from using some kind of pseudo enum rather than a raw boolean.

`updateOnboardingInitial(false)` is ambiguous, while `updateOnboardingInitial(Steps.FIRST_STEP);` may provide more clarity.

In-general, passing raw booleans is confusing->
```
makeButton(true); // What does true do??

const isFancyButton = true;
makeButton(isFancyButton);
```